### ISSUE TITLE
Multi-line featured resources are not indented properly

### ISSUE THREAD

**Issue by juliandsmith** [AASHE/hub#265]
When a featured resource with long titles requires more than one line, the second line should start directly below the first line (as in list with bullets). Currently, the second line starts under the star, which doesn't look great:
![image](https://cloud.githubusercontent.com/assets/15839046/14107678/971664ca-f587-11e5-85ad-45eff299f7f4.png)

**Comment by dodget** [comment]
@juliandsmith - I began messing around with this today. I found the whole 'Featured Resource' panel to be a little crowded and clunky (especially when a user is not logged in). I've separated each resource instance into three buckets: Star or Lock, Title, Content Type Label. They wont overlap now. I've added a little front end logic to check if the user should see a star, or a lock, but they should never see both for a single resource listing. I've attached screen shots of when a user is logged in, and when they aren't. I know that I'll have to tweak some alignments, but let me know what you think!

Logged in:
<img width="766" alt="logged-in" src="https://user-images.githubusercontent.com/12410482/33787298-9bed3b0e-dc3a-11e7-8ac3-928b60c77628.png">

Logged out, need to align the star and lock:
<img width="764" alt="logged-out" src="https://user-images.githubusercontent.com/12410482/33787319-b32e8fde-dc3a-11e7-8023-02761d8d2f8e.png">
